public/account.html

```diff
@@ -83,7 +83,9 @@
 
           <p>
             <button class="primary" type="submit">Send</button>
-            <a href="#" data-action="show-signin">Cancel</a>
+            <small>
```

Comment:
Was able to understand your comment. I forgot to apply `<small>` to these things before but every action next to a button has the `<small>` applied to it, it makes it look more visually balanced that the secondary action font size isn't as big as the primary button to the left of it.

But if this causes any problem I can remove it!